common/containers/Tabs/Contracts/components/Interact/components/InteractForm/index.tsx

```diff
@@ -69,6 +71,16 @@ class InteractForm extends Component<Props, State> {
     if (nextProps.currentTo.raw !== prevProps.currentTo.raw) {
       nextProps.resetState();
     }
+
+    if (nextProps.currentUnit !== prevProps.currentUnit && nextProps.currentUnit) {
```

Comment:
I need help to trigger `accessContract`. I can't do it after `handleInput` or `UNSAFE_componentWillReceiveProps` inside the new condition.